pnpm-workspace.yaml

```diff
@@ -22,4 +22,4 @@ catalog:
   "@aws-sdk/client-dynamodb": ^3.879.0
   "@aws-sdk/util-dynamodb": ^3.879.0
   "@guardian/prettier": ^2.1.5
-  "@guardian/support-service-lambdas": "guardian/support-service-lambdas#0d59cdc60cfd25edf626badfd01a55b1aa4a7c41"
+  "@guardian/support-service-lambdas": "guardian/support-service-lambdas#bfcc49982d868e6d84fc7c379d01c5b03946d793"
```

Comment:
Don't forget to merge guardian/support-service-lambdas#3150 before merging this PR (and update the SHA).